### PR TITLE
gh-128062: Fix the font size and shortcut display of the turtledemo menu

### PR DESCRIPTION
**Pull request by Xiaokang2022**
Leave the font of the menu bar the default to keep it consistent with the rest of the world.  Display the shortcut keys in the right way, using the 'accelerator' option.

* Issue: gh-128062

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2024-12-18-10-18-55.gh-issue-128062.E9oU7-.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Improve the menu bar of ``turtledemo``.
```
Comment (ZeroIntensity) [inline Misc/NEWS.d/next/Library/2024-12-18-10-18-55.gh-issue-128062.E9oU7-.rst]:
```suggestion
Improve the menu bar of :mod:`turtledemo`.
```

**Review by ZeroIntensity — COMMENTED** [review]
Thanks! Could we be more clear (in both the PR title and the blurb entry) about what this is changing? ("improve" is too vague of a term)

**Comment by terryjreedy** [comment]
I fixed title and commit message.  The macOS failure seems to be a known unrelated issue, #128069.  Merging seems to be blocked until this is fixed.

**Review by terryjreedy — APPROVED** [review]
On my MacBook Air, with a small screen, fontsize 12 is too small as the default, used for the top bar, appears to be 14.  Command+- and Command+= are the keys that work in the text widget.  Agree with leaving size alone and using the standard accelerator option.

**Pull request merged**